rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -91,6 +91,16 @@ object Jupyter {
                             Some(xbounds))
     })
 
+  def whiskers[K, N](samples: Seq[Map[String, Double]]): Plot = {
```

Comment:
It looks like the type parameters here aren't used. Should they be in place of String and Double?

Reply:
that was from an earlier version and I forgot to clean them up. The types in this currently exactly match `precis`, which seems appropriate. We may in a later PR want to make both of those methods more flexible.